hydromt_wflow/wflow.py

```diff
@@ -1964,22 +1965,27 @@ def write_forcing(
                         times = da.time.values
                     if start < pd.to_datetime(times[0]):
                         start = pd.to_datetime(times[0])
-                        missings = True
+                        correct_times = True
+                    elif start not in times:
+                        start = pd.to_datetime(times[times >= start][0])
```

Comment:
I think if starttime not in forcing_fn, the code here should be:
start = pd.to_datetime(times[0])
to make the starttime similar to the starttime of forcing_fn

times[times >= start][0]
the above line would result in an error as start is not in times. 

I will commit this change

Reply:
actually the same issue occurred if end time was lower than times[-1] (so also not in times), so unless I overlooked somehting, I think the code can be simplified to:

```
                    if (start < pd.to_datetime(times[0])) | (start not in times):
                        start = pd.to_datetime(times[0])
                        correct_times = True
                    if (end > pd.to_datetime(times[-1])) | (end not in times):
                        end = pd.to_datetime(times[-1])
                        correct_times = True
```

this is done in commit 6c4f1317e207e04355ad4ed3ce6c763ae095a6c1